dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/logging/LogUtils.java

```
@@ -264,26 +262,45 @@ protected static Logger createLogger(final Class<?> cls,
                 try {
                     b = BundleUtils.getBundle(cls, bundleName);
                 } catch (MissingResourceException rex) {
-                    //ignore
```

Can we log ignored exception here?